### ISSUE TITLE
Edit Page V2: Blank Page When You Edit a Contentlet

### ISSUE THREAD

**Issue by fmontes** [dotCMS/core#27900]
### Parent
#27546 

### Problem Statement
When editing content within a page using the new page editor, the page becomes blank and does not display any content.

### Steps to Reproduce

https://github.com/dotCMS/core/assets/751424/19c8f6f2-152d-412b-8d25-602562c415e1

```[tasklist]
### Acceptance Criteria
- [ ] When edit a contentlet, the page should refresh and display the updated content correctly.
```

### External Links
- None

### Assumptions & Initiation Needs
N/A

### Quality Assurance Notes & Workarounds
N/A

**Comment by fmontes** [comment]
Fixed but found this #27964

**Comment by josemejias11** [comment]
**Approved**: Tested on **master_ca3eb44**, Docker, macOS 13.0, FF v121.0.1
